runtime/stout/src/lib.rs

```diff
@@ -444,7 +444,7 @@ impl pallet_uniques::Config for Runtime {
 }
 
 parameter_types! {
-	pub MaximumSchedulerWeight: Weight = Weight::from_ref_time(10_000_000);
+	pub MaximumSchedulerWeight: Weight = Weight::from_parts(10_000_000, 0);
```

Comment:
this is an important catch. what do you guys think if we substitute it with 
```
pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * BlockWeights::get().max_block;
```
like it is done in [polkadot](https://github.com/paritytech/polkadot/blob/9420d7dc222ff9490625c108ad9011fd825065f0/runtime/polkadot/src/lib.rs#L181) and [substrate](https://github.com/paritytech/substrate/blob/bbe132e2ebf9ee919c6045271d5bde27bace5659/bin/node/runtime/src/lib.rs#L358), since `10_000_000` is also very limiting. Or is it done this way for a reason?

`u64::MAX` is probably too much since the [max allowed POV](https://github.com/paritytech/polkadot/blob/95fe4c8862810bffd68343231a517e62689c05c0/primitives/src/v2/mod.rs#L348) is currently 5 mb

Reply:
I think that's a good solution. Does anyone know the reason why its 80%?